rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -246,6 +280,29 @@ def _decode_elfi_claim(self, context: DecoderContext) -> DecodingOutput:
 
         return DEFAULT_DECODING_OUTPUT
 
+    def _decode_ens_claim(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] != ENS_CLAIM:
+            return DEFAULT_DECODING_OUTPUT
+
+        raw_amount = hex_or_bytes_to_int(context.tx_log.data[:32])
+        amount = asset_normalized_value(amount=raw_amount, asset=self.ens)
```

Comment:
nitpick: if we don't need the it anywhere else, can use decimals here and avoid a resolution and keeping it as self.ens, since we can just use A_ENS. Same for some other parts of this module